Classes/Emogrifier.php

```diff
@@ -671,6 +671,7 @@ private function copyCssWithMediaToStyleNode(\DOMDocument $xmlDocument, \DOMXPat
             foreach ($this->parseSelectors($mediaQuery['css']) as $selector) {
                 if ($this->existsMatchForCssSelector($xpath, $selector['selector'])) {
                     $mediaQueriesRelevantForDocument[] = $mediaQuery['query'];
+                    break;
```

Comment:
Ah, okay. Could you please move this to a separate PR (including two unit tests: one that tests that the bug is fixed, and one that tests that multiple queries still work)?